src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -76,6 +81,9 @@ const Ownership = () => {
     [getOwnershipRequestsQuery.data],
   );
 
+  // @ts-expect-error
+  const creator = getOwnershipCreatorQuery.data;
```

Comment:
```suggestion
  const creator: OwnershipCreator | undefined = getOwnershipCreatorQuery.data;
```
I think you can manually add the type

Reply:
The problem is that the type does not include a data property.